roles/common/tasks/pod_tests.yml

```
@@ -0,0 +1,20 @@
+---
+
+- name: Get Pod Instance "{{ pod_status_str }}" 
+  ansible.builtin.shell:
+    cmd: |
+      oc get pods -n  "{{ pod_nspace }}" | grep "{{ item }}" | grep "{{ pod_status_str }}" | awk '{print $1;}'
+  register: podinstance
+  changed_when: false
+
+- name: Check terminated pod 
+  ansible.builtin.shell:
+    cmd: | 
+      oc get pod -n "{{ pod_nspace }} {{ podinstance.stdout }}"
```

```suggestion
      oc get pod -n "{{ pod_nspace }}" "{{ podinstance.stdout }}"
```

Please don't apply these suggestions, they are reflective of what is going on in PR#149